lib/apiservers/engine/backends/container.go

```diff
@@ -58,6 +58,19 @@ import (
 
 const (
 	bridgeIfaceName = "bridge"
+
+	// MemoryMinMiB - the minimum allowable container memory size
+	MemoryMinMiB = 1024
```

Comment:
The minimum should be associated with the bootstrap image that's in use - just a note for the future - but regardless is not something I'd considered user configurable.